ann_benchmarks/datasets.py

```diff
@@ -69,10 +72,49 @@ def write_output(train, test, fn, distance, point_type='float', count=100):
         distances[i] = [d for _, d in res]
     f.close()
 
+"""
+param: train and test are arrays of arrays of indices.
+"""
+def write_sparse_output(train, test, fn, distance, dimension, count=100):
+    from ann_benchmarks.algorithms.bruteforce import BruteForceBLAS
+    f = h5py.File(fn, 'w')
+    f.attrs['type'] = 'sparse'
+    f.attrs['distance'] = distance
+    f.attrs['dimension'] = dimension
+    f.attrs['point_type'] = 'bit'
+    print('train size: %9d * %4d' % (train.shape[0], dimension))
+    print('test size:  %9d * %4d' % (test.shape[0], dimension))
+
+    train = numpy.array(list(map(sorted, train)))
+    test = numpy.array(list(map(sorted, test)))
```

Comment:
I think we need to add somewhere in the documentation that elements in sets are guaranteed to be ordered. I think this can speed up some filtering approaches.

Reply:
I added a note in the README in replacement of the former one about no longer supporting set similarity. Is this what you had in mind?

Reply:
I think that's good, yes.